src/functions/utils/excelApi.ts

```diff
@@ -0,0 +1,82 @@
+/* Copyright */
+import { getCachedAccessToken } from "./accessToken";
+
+const CACHE_EXPIRATION_TIME = Date.now() + 5 * 60 * 1000;
```

Comment:
This will be run at the file initialization and will not change after that. If the lambda environment stays up, this will stay the same forever. Make this into a function as in the accessToken.ts files so that it is refreshed on each call. Furthermore, you could make it so that only the hard-coded numbers are in the const. The Date.now() function could be the callers responsibility. Same thing could be changed in accessToken too.